malariagen_data/af1.py

```diff
@@ -148,7 +146,10 @@ def __init__(
             show_progress=show_progress,
             check_location=check_location,
             pre=pre,
-            **kwargs,  # used by simplecache, init_filesystem(url, **kwargs)
+            gcs_url=GCS_URL,
+            major_version_number=MAJOR_VERSION_INT,
+            major_version_path=MAJOR_VERSION_GCS_STR,
```

Comment:
Making these parameters to the super-class constructor makes testing much easier.